unpick-cli/src/main/java/daomephsta/unpick/cli/Main.java

```diff
@@ -75,11 +81,8 @@ private static void unpick(Path inputJar, Path outputJar, Path unpickDefinition,
                     InputStream inputStream = jarFile.getInputStream(entry);
 
                     if (entry.getName().endsWith(".class")) {
-                        ClassReader classReader = new ClassReader(inputStream);
-                        ClassNode classNode = new ClassNode();
-                        classReader.accept(classNode, 0);
-
-                        uninliner.transform(classNode);
+                        String internalName = entry.getName().substring(0, entry.getName().length() - ".class".length());
```

Comment:
I'm unsure if this correctly converts from ZIP entry path to class internal name. I would appreciate a second opinion.